src/Service/CallService.php

```diff
@@ -227,7 +228,7 @@ public function call(
         array $config = [],
         bool $asynchronous = false,
         bool $createCertificates = true
-    ): Response {
+    ) {
```

Comment:
Will this ever by anything else than this type of Response?
_It feels weird to remove this, would like to see all possible types this function could return_

Reply:
Yes, in federated calls it will return Proxy.
Agreed on it feeling weird, but I can only add it when rebuilding to PHP 8.2.